roles/kubernetes/master/templates/kubeadm-config.v1alpha1.yaml.j2

```diff
@@ -149,6 +167,12 @@ apiServerExtraVolumes:
   mountPath: {{ kube_token_dir }}
 {% endif %}
 {% endif %}
+{% if (cloud_provider is defined and cloud_provider in ["openstack", "azure", "vsphere"]) or volume_cross_zone_attachment %}
+schedulerExtraVolumes:
+- name: cloud-config
```

Comment:
delete it ,Cloud provider need to configure apiserver、controllerManager and kubelet. not kube-scheduler.